dhis-2/dhis-support/dhis-support-system/src/main/java/org/hisp/dhis/system/grid/ListGrid.java

```diff
@@ -309,7 +309,13 @@ public List<GridHeader> getMetadataHeaders()
     @Override
     public int getIndexOfHeader( String name )
     {
-        return headers.indexOf( new GridHeader( name, null ) );
+        return headers.indexOf( new GridHeader( name ) );
+    }
+
+    @Override
+    public boolean headerExists( String name )
```

Comment:
This could have been a `default` method in the interface implemented like this to avoid having to reimplement this e.g. in testing mocks etcetera.

Reply:
Okay will update.